apps/desktop/tests/services/AgentService/contracts/errors-i18n.test.ts

```
@@ -48,16 +48,34 @@ describe('AiError display localization', () => {
             'No endpoints found that support tool, image, and file inputs'
         );
 
-        expect(error.getDisplayMessage()).toBe('当前模型不支持图片/文件输入，请选择合适模型继续。');
+        expect(error.getDisplayMessage()).toBe(
+            AiError.getMessage(AiErrorCode.UNSUPPORTED_INPUT)
+        );
```

ESLint

[error] Replace `⏎············AiError.getMessage(AiErrorCode.UNSUPPORTED_INPUT)⏎········` with `AiError.getMessage(AiErrorCode.UNSUPPORTED_INPUT)`

(prettier/prettier)